libmcount/dynamic.c

```diff
@@ -73,6 +74,39 @@ static struct mcount_orig_insn *lookup_code(struct rb_root *root,
 	return iter;
 }
 
+static inline bool frozen(struct code_page *cp)
+{
+	return cp->frozen;
+}
+
+static inline int release(struct code_page *cp)
+{
+	int res = 0;
```

Comment:
no need to initialize.

Reply:
it was changed to use list. this code does not exist.